gatsby/sourceplugins.rst

```diff
@@ -3,33 +3,30 @@ Source plugins
 
 Now that we learned how to create pages and access dynamic data with GraphQL, we can start fetching some data from external sources.
 
-Source plugins are the tools that can do this job. Every plugin is specialized in getting data from a different source.
+Source plugins are the tools that can handle this job. Every plugin is specialized in getting data from a different source.
 
-..note :: There are a lot of plugins for almost every need. In GatsbyJS website, you can find a complete list of `official plugins <https://www.gatsbyjs.org/plugins/>`_.
+.. note::
 
-Let's start with a basic one: `gatsby-source-filesystem` a plugin that transforms files in GraphQL nodes.
+  There are a lot of plugins for almost every need. You can find a complete list of plugins on `GatsbyJS official website <https://www.gatsbyjs.org/plugins/>`_.
+
+Let's start with a basic one, ``gatsby-source-filesystem``: a plugin that transforms files into GraphQL nodes.
 
 .. code-block:: none
   
     npm install --save gatsby-source-filesystem
 
-After that, we need to enable the plugin in our project. To do this, we need to add it into `gatsby-config.js` file.
+After that, we need to enable the plugin in our project.
 
-.. code-block:: none
+To do this, we need to add it into ``gatsby-config.js`` file.
```

Comment:
Above 2 sentences can be 1 paragraph by removing blank line that separates them.